### ISSUE TITLE
Update torii dependency to latest version

### ISSUE THREAD

**Issue by marcoow** [simplabs/ember-cli-simple-auth-torii#11]
see #10

**Comment by Frozenfire92** [comment]
I'm going to tackle this one

**Comment by dbachet** [comment]
Hi, as I understand, currently I need to use the master branch of `ember-cli-simple-auth-torii` to have torii to the latest version as a dependency. 

I followed https://github.com/ember-cli/ember-cli#working-with-master to use master branch for an addon. 

So I did clone `ember-cli-simple-auth-torii` and `npm link` + `npm link ember-cli-simple-auth-torii`. But when I `npm ls|grep -i torii` I'm still getting 
```
├─┬ ember-cli-simple-auth-torii@0.7.3
│ └── torii@0.1.3
```

There should be `0.3.2`, isn't it?

Anyone would be able to help me?

Thx
